depth_to_pointcloud/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>depth_to_pointcloud</name>
+  <version>0.0.0</version>
+  <description>A simple node to convert a depth image and camera info into a point cloud.</description>
```

Comment:
I just changed the description for now.  If you really think we should, I can change the name of the whole package, but I'd rather not.